r/tests/testthat/test-RecordBatch.R

```diff
@@ -513,6 +513,76 @@ test_that("record_batch() with different length arrays", {
   expect_error(record_batch(a = 1:5, b = 1:6), msg)
 })
 
+test_that("RecordBatch doesn't support rbind", {
+  expect_snapshot_error(
```

Comment:
Not a critique of this PR, just an observation: I find these `expect_snapshot_error` tests difficult as a reader. It's really not clear what they're doing, you have to look aside to another file and apparently count the number of errors in the file to see what message this particular call threw.

Reply:
On the plus side, they make it really easy to capture the whole error we test against and show that in the PR diff. Dewey suggested we add this when earlier in this PR we had found the errors were passing my regex test but were actually wrong.

> apparently count the number of errors in the file to see what message this particular call threw.

They are labelled with what test they are in, so you should only have to count within a test (if there even are multiple snapshots per a test.)